src/app/components/notes/notes.service.ts

```diff
@@ -7,32 +7,27 @@ export class NotesService {
   [x: string]: any;
```

Comment:
Why do we need to do this?

Reply:
I noticed it while reviewing before pushing and wasn't sure if it was added by TypeScript or if I pasted it while trying to solve a bug. Looks like it doesn't do anything, I'll remove it.